src/codebook/controls/filters/update.js

```diff
@@ -76,17 +77,18 @@ export function update(codebook) {
     .attr('selected', d => (d.selected ? 'selected' : null));
 
   //Initialize event listeners
-  filterCustom.on('change', function() {
-    //display the loading indicator
-    codebook.loadingIndicator.style('display', 'block');
-    //wait by the quarter second until the loading indicator is visible to re-render everything
-    const loading = setInterval(() => {
-      const display = codebook.loadingIndicator.style('display');
-      if (display === 'block') {
+  var filters = codebook.controls.wrap
+    .selectAll('.filterCustom select')
+    .on('change', function(d) {
+      indicateLoading(codebook, '#loading-indicator', () => {
```

Comment:
This seems tautological (but it does seem to work). Doesn't this mean to show the `#loading-indicator` spinner until the `#loading-indicator` spinner appears?!